pkg/state/state_nv_peer.go

```diff
@@ -103,6 +108,13 @@ func (s *stateNVPeer) Sync(customResource interface{}, infoCatalog InfoCatalog)
 	if err != nil {
 		return SyncStateNotReady, errors.Wrap(err, "failed to create/update objects")
 	}
+
+	if s.shouldIgnoreStatus(nodeInfo) {
```

Comment:
nit: can you add a short one-liner comment why this is called.
(i know that this method description explains it, but a gist as a reminder here would be helpful to avoid looking it up in where the method is defined is helpful.)